main/FatesRestartInterfaceMod.F90

```diff
@@ -3160,9 +3184,25 @@ subroutine get_restart_vectors(this, nc, nsites, sites)
              end do
           end do
 
+          do i_scls = 1, nlevsclass
+             do i_pft = 1, numpft
+                do i_term_type = 1, n_term_mort_types
+                   sites(s)%term_nindivs_canopy(i_term_type,i_scls,i_pft) = rio_termnindiv_cano_siscpf(io_idx_si_scpf_term)
+                   sites(s)%term_nindivs_ustory(i_term_type,i_scls,i_pft) = rio_termnindiv_usto_siscpf(io_idx_si_scpf_term)
+                   io_idx_si_scpf_term = io_idx_si_scpf_term + 1
+                end do
```

Comment:
Similar to the above section in `set_restart_vectors`, I believe this could be nested into the sizeclass and pft loop above.